docs/developer-guide/syntax-reference.rst

```diff
@@ -1912,6 +1912,12 @@ Scalar functions
 | REPLACE                |  ``REPLACE(col1, 'foo', 'bar')``                                          | Replace all instances of a substring in a string  |
 |                        |                                                                           | with a new string.                                |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| REGEXP_EXTRACT         |  `REGEXP_EXTRACT('.*', col1)` or                                          | Extract the first subtring matched by the regex   |
```

Comment:
I know this is confusing, but the .rst docs are deprecated going forward, and we're doing all new content in the docs-md directory, Can you please move this content to the the [Scalar Functions topic](https://github.com/confluentinc/ksql/blob/master/docs-md/developer-guide/ksqldb-reference/scalar-functions.md)? Thanks!

Reply:
@JimGalasyn so I should just get rid of this file from the PR then?